src/components/chat-view-container/chat-view-container.tsx

```diff
@@ -96,22 +97,27 @@ export class Container extends React.Component<Properties, State> {
   componentDidMount() {
     const { channelId } = this.props;
     if (channelId) {
+      // only set the active channel ID if it's not a direct message conversation
+      if (!this.props.activeConversationId) {
```

Comment:
Wouldn't this be set if you had both channels and the chat window open? So, if you have the chat window open we wouldn't change the activeChannelId correctly?

Also, do we clear the activeChannelId somewhere when we navigate away from the channels route?

I think maybe managing the activeChannelId could be it's own PR so we can figure out the details there first.

Reply:
Actually, maybe this should be one or two steps higher where we're specifically rendering the Channel. Eg: the `platform-app/channels/container.tsx` component.

Reply:
created a separate PR for it